client/src/modules/list/model/actions.js

```diff
@@ -375,7 +379,7 @@ export const addListViewer = (listId, email) => dispatch =>
     email
   })
     .then(resp => {
-      if (resp.status === 200) {
+      if (resp.status === ServerCode.OK) {
```

Comment:
Not clear what you mean.

From backend I'm returning two diffrent statutes so I need to check it here, 

if I add this piece of logic into `handleFetchErrors` function, the application will stop working, because all actions are handling resp.json() by itself. So if resp.json() should be here like this

```
if(res.status === ServerCode.OK) {
     return resp.json()
}
```
**then all actions need to be reviewed and changed..**

```

const handleFetchErrors = resp => {
  if (resp.status === ServerCode.FORBIDDEN) {
    setTimeout(() => {
      window.location = '/';
    }, NOTIFICATION_TIMEOUT);
  }

  if (resp.status === ServerCode.NOT_FOUND) {
    history.replace('/page-not-found');
    throw new Error();
  }

  if (resp.status >= ServerCode.BAD_REQUEST && resp.status < 600) {
    throw new Error();
  }

  return resp;
};
```